lib/experimental/new_embed.dart

```diff
@@ -1093,43 +1131,88 @@ class ConsoleExpandController extends ConsoleController {
 enum DialogResult {
   yes,
   no,
+  ok,
+  cancel,
 }
 
 class Dialog {
   final MDCDialog _mdcDialog;
-  final Element _yesButton;
-  final Element _noButton;
+  final Element _leftButton;
+  final Element _rightButton;
   final Element _title;
   final Element _content;
 
   Dialog()
       : _mdcDialog = MDCDialog(querySelector('.mdc-dialog')),
-        _yesButton = querySelector('#dialog-yes'),
-        _noButton = querySelector('#dialog-no'),
+        _leftButton = querySelector('#dialog-left-button'),
+        _rightButton = querySelector('#dialog-right-button'),
         _title = querySelector('#my-dialog-title'),
         _content = querySelector('#my-dialog-content');
 
-  Future<DialogResult> show(String title, String htmlMessage,
+  Future<DialogResult> showYesNo(String title, String htmlMessage,
       {String yesText = "Yes", String noText = "No"}) {
+    return _setUpAndDisplay(
+        title, htmlMessage, yesText, noText, DialogResult.yes, DialogResult.no);
+  }
+
+  Future<DialogResult> showOk(String title, String htmlMessage) {
+    return _setUpAndDisplay(
+      title,
+      htmlMessage,
+      "OK",
+      '',
+      DialogResult.ok,
+      DialogResult.cancel,
```

Comment:
The Material guidelines always recommend the dismissive action be on the left: https://material.io/design/components/dialogs.html#actions

more discussion about this here:
https://www.nngroup.com/articles/ok-cancel-or-cancel-ok/

Reply:
You clearly care about this way more than I do. 😄 

Matching the spec is a good idea. I've updated the code.